src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -2,28 +2,43 @@
 import React, {
   useRef, useEffect, useState, useCallback,
 } from 'react';
+import dynamic from 'next/dynamic';
 import PropTypes from 'prop-types';
 import { useSelector, useDispatch } from 'react-redux';
 import {
   Empty, Typography, Skeleton,
 } from 'antd';
 import _ from 'lodash';
+
 import { getCellSets, getCellSetsHierarchyByKeys } from 'redux/selectors';
 import calculateIdealNMarkerGenes from 'utils/calculateIdealNMarkerGenes';
-import spec from '../../../utils/heatmapSpec';
-import VegaHeatmap from './VegaHeatmap';
-import PlatformError from '../../PlatformError';
-import { updateCellInfo } from '../../../redux/actions/cellInfo';
-import { loadGeneExpression, loadMarkerGenes } from '../../../redux/actions/genes';
-import { loadCellSets } from '../../../redux/actions/cellSets';
 
-import { loadComponentConfig } from '../../../redux/actions/componentConfig';
-import populateHeatmapData from '../../plots/helpers/populateHeatmapData';
-import Loader from '../../Loader';
+import { loadCellSets } from 'redux/actions/cellSets';
+import { loadGeneExpression, loadMarkerGenes } from 'redux/actions/genes';
```

Comment:
Can you check if these imports are correct? I am getting a warning when I open them on my vs code:
<img width="1051" alt="Screenshot 2021-12-02 at 15 41 25" src="https://user-images.githubusercontent.com/65414651/144443616-b33b87d2-11d5-4474-9a9b-6580c469c69e.png">

Reply:
![image](https://user-images.githubusercontent.com/8465903/144666517-f51d478b-2c21-4b60-9cdc-96884b9fefed.png)
I'm not getting that warning, perhaps an `npm ci` fixes your setup?

Reply:
hmm, it still didn't ...

Reply:
Can I resolve this? I don't know what is the problem there, but it seems like a specific issue, not of this PR

Reply:
yes, let's resolve this